source/whats-new.txt

```diff
@@ -82,6 +82,76 @@ The 3.0 driver release includes the following new features:
   `TimeOnly Struct. <https://learn.microsoft.com/en-us/dotnet/api/system.timeonly?view=net-6.0>`__
   API reference page on MSDN.
 
+- Adds support for client-side projection when using the ``Find()`` method,
+  the ``Select()`` method, or the ``Project()`` aggregation stage.
+  In previous versions of the driver, you could perform client-side
+  projection only after calling the ``ToEnumerable()`` or ``AsEnumerable()`` method, and
+  only within the ``Select()`` method.
```

Comment:
In previous versions of the driver behavior depended on which LINQ provider you were using:

* LINQ2: implicit client-side projections in `Find`, `Project` and `Select`
* LINQ3: explicit client-side projections only

In the 3.0 version of the driver implicit client-side projections are supported but only if `EnableClientSideProjections == true`

By "explicit client-side projections" I mean calling `ToEnumerable()` or `AsEnumerable()` to specify where the client-side part of the query starts.

Reply:
Thanks. I reworded this section. Is it accurate to say that v3.0 supports _implicit_ client-side projection when using the LINQ3 provider, so long as that setting is true?